silverback/_cli.py

```diff
@@ -425,6 +505,12 @@ def bots():
 @click.option("-n", "--network", required=True)
 @click.option("-a", "--account")
 @click.option("-g", "--group", "vargroups", multiple=True)
+@click.option(
+    "-r",
+    "--registry-credentials",
+    "registry_credentials_name",
+    help="registry credentials to use to pull the image",
+)
 @click.argument("name")
 @cluster_client
 def new_bot(
```

Comment:
Is there a way to also add or remove (or modify) credentials when updating a bot as well?

Reply:
I'm not sure what you mean? Like during the process of adding a bot? They're separate entities and are administered separately in the usual CRUD form.

Reply:
no I mean I think this option is missing from `update_bot`, but also there you would have to consider if you wanted it to stay the same, add a new one, replace one that was there, or delete an existing credential to use for the bot's definition

Reply:
Ah, missing the CLI command. Correct, will fix.

Reply:
Done in [b37334d](https://github.com/ApeWorX/silverback/pull/120/commits/b37334d424d1d2c9376b1c0d90fd328eed22fd72)